Common/Domain/Domain.fsproj

```diff
@@ -24,15 +25,21 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <WarningLevel>3</WarningLevel>
     <OtherFlags>--keyfile:..\..\tddstud10.snk</OtherFlags>
+    <RunCodeAnalysis>False</RunCodeAnalysis>

```

Comment:
I'll revert changes to the F# project files as you suggested. I naively assumed that uninstalling the FxCop NuGet reference would get things back to the way they were before